specification/logic/resource-manager/Microsoft.Logic/stable/2016-06-01/logic.json

```diff
@@ -1636,24 +1796,24 @@
         ],
         "responses": {
           "200": {
-            "description": "OK"
-          },
-          "204": {
-            "description": "No Content"
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/WorkflowRunActionRepetitionDefinition"
+            }
           }
         }
       }
     },
-    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Logic/integrationAccounts/{integrationAccountName}/listCallbackUrl": {
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Logic/workflows/{workflowName}/runs/{runName}/actions/{actionName}/repetitions/{repetitionName}/listExpressionTraces": {
       "post": {
         "tags": [
-          "IntegrationAccounts"
+          "WorkflowRunActions"
         ],
-        "operationId": "IntegrationAccounts_GetCallbackUrl",
```

Comment:
It seems that this operation used to be incorrectly named which you've fixed with the PR - is that correct?
Just so you're aware - this will be a breaking change - and therefore major version bump for the generated clients.

Reply:
Yeah, it was the odd one out. I was really hoping that it wouldn't entail a breaking change. I'll revert it.